linkerd/stack/metrics/src/service.rs

```diff
@@ -13,15 +13,30 @@ use tokio::time::Instant;
 pub struct TrackService<S> {
     inner: S,
     metrics: Arc<Metrics>,
-    blocked_since: Option<Instant>,
+    blocked_since: Option<time::Instant>,
+    _tracker: Arc<()>,
```

Comment:
I briefly wondered why we couldn't use the `Arc<Metrics>` reference count and had to introduce a second `Arc`, but I realized that the `Metrics` is not just shared with all clones of this specific service instance. It _might_ be worth adding a comment here explaining that?